pkgs/applications/window-managers/wayfire/wayfire-plugins-extra.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wayfire
+, wf-config
+, gtkmm3
+, gtk-layer-shell
+, libxkbcommon
+, xorg
+}:
+
+stdenv.mkDerivation rec {
```

Please use `finalAttrs` format:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```